images/linux/scripts/base/apt.sh

```diff
@@ -20,10 +20,8 @@ echo "APT::Get::Assume-Yes \"true\";" > /etc/apt/apt.conf.d/90assumeyes
 # Uninstall unattended-upgrades
 apt-get remove unattended-upgrades
 
-# Install aria2 and jq
-apt-get install aria2
-apt-get install jq
 # Use apt-fast for parallel downloads
 add-apt-repository -y ppa:apt-fast/stable
 apt-get update
-apt-get install apt-fast
+# Install aria2 ,jq and apt-fast
+apt-get install  aria2 jq apt-fast
```

Comment:
There is an extra space between `install` and `aria2`. Minor issue that could be resolved after merging. Just something to keep an eye out for the future.